dragonfly/block/glowstone.go

```diff
@@ -0,0 +1,38 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"math/rand"
+)
+
+// Glowstone is commonly found on the ceiling of the nether dimension.
+type Glowstone struct{ noNBT }
```

Comment:
Pretty sure there must be a space after struct

Reply:
For single line structs I think this is how Go formats it.